src/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,70 @@
+import classNames from 'classnames';
+import React from 'react';
+import styles from './Pagination.module.scss';
+
+const LAST_PAGE_NUMBER = 18;
+
+type Props = {
+  currentPage: number;
+  setCurrentPage: (prevValue: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  currentPage,
+  setCurrentPage,
+}) => {
+  const getPages = (pagesNumber: number) => {
```

Comment:
move this function outside of the Pagination component as it's static and does not depend on any state/props